### DISCUSSION TITLE
Is it possible to inject the csrf token into forms?

### DISCUSSION THREAD

**Discussion by dickermoshe** [reactive-python/reactpy-django#257] [Problem] [answered]
Seems that there is no way to do this. Is this the case?

**Comment by Archmonger** [comment, marked as answer]
CSRF can be injected as a hidden input. See [this answer](https://discord.com/channels/1111078259854168116/1150465007327396052/1150648843726225489) on Discord.

**Reply by dickermoshe** [reply]
Very smart. This is one of the coolest packages I've ever seen.

Everyone should copy this approach. While I wish there was something like jsx for python, the underlying architecture here is perfect.
The smartest way I've ever seen to keep all state on the server and still have great performance.

I was concerned about this packages health, but getting an answer so quickly is very reassuring.
